simtools/simtel/simtel_runner.py

```diff
@@ -173,13 +181,25 @@ def run(self, test=False, force=False, input_file=None, run_number=None):
 
     @staticmethod
     def _simtel_failed(sys_output):
+        """Test if output is different than 0
+
+        Returns
+        -------
+        bool
+            1 if sys_output is different than 0, and 1 otherwise.
+        """
         return sys_output != 0
 
     def _raise_simtel_error(self):
         """
-        Raise sim_telarray execution error. Final 30 lines from the log file
-        are collected and printed.
+        Raise sim_telarray execution error. Final 30 lines from the log file are collected and
+        printed.
```

Comment:
Do we need a backslash here to connect lines?

Reply:
For comments it does seem to be necessary but for consistency I am adding one here